packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1045,6 +1045,12 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   bool _drawerOpened = false;
   bool _endDrawerOpened = false;
 
+  /// Whether the [Scaffold.drawer] is opened.
```

Comment:
A "see also" section that pointed at https://docs.flutter.io/flutter/material/ScaffoldState/openDrawer.html would be helpful.

The openDrawer() doc should similarly "see also" isDrawerOpen.